ouroboros-consensus-cardano/tools/db-analyser/Block/Cardano.hs

```diff
@@ -56,6 +64,37 @@ analyseBlock f =
     p :: Proxy HasAnalysis
     p = Proxy
 
+analyseWithLedgerState ::
```

Comment:
Please add a comment: the goal is just to apply the given analysis to the current ledger state, right?